1.regression_modelling/scripts/1.train_models.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Train ElasticNet model to predict each A647 and GOLD feature using the nuclear features
+# 
+# In this notebook, we split the features into each group for nuclear speckle or nucleus and then train a model per nuclear speckle feature using the nuclear features to predict it.
```

Comment:
Might be worth specifying the model type here if it is a classifier or a regression model.

Reply:
Added!